prisma/schema.prisma

```diff
@@ -2,108 +2,111 @@
 model Account {
-    id                 String  @id @default(auto()) @map("_id") @db.ObjectId
-    userId             String  @db.ObjectId
-    type               String
-    provider           String
-    providerAccountId  String
-    refresh_token      String? @db.String
-    access_token       String? @db.String
-
-    // Unix timestamp
-    // @todo(nickbar01234) - I actually couldn't find a document that said this definitely, but this assumption made 
-    // sense if you convert the value to regular time 
-    expires_at         Int?
-
-    token_type         String?
-    scope              String?
-    id_token           String? @db.String
-    session_state      String?
-    oauth_token_secret String?
-    oauth_token        String?
-
-    user User @relation(fields: [userId], references: [id], onDelete: Cascade)
-
-    @@unique([provider, providerAccountId])
+  id                String  @id @default(auto()) @map("_id") @db.ObjectId
+  userId            String  @db.ObjectId
+  type              String
+  provider          String
+  providerAccountId String
+  refresh_token     String? @db.String
+  access_token      String? @db.String
+
+  // Unix timestamp
+  // @todo(nickbar01234) - I actually couldn't find a document that said this definitely, but this assumption made 
+  // sense if you convert the value to regular time 
+  expires_at Int?
+
+  token_type         String?
+  scope              String?
+  id_token           String? @db.String
+  session_state      String?
+  oauth_token_secret String?
+  oauth_token        String?
+
+  user User @relation(fields: [userId], references: [id], onDelete: Cascade)
+
+  @@unique([provider, providerAccountId])
 }
 
 model Session {
-    id           String   @id @default(auto()) @map("_id") @db.ObjectId
-    sessionToken String   @unique
-    userId       String   @db.ObjectId
-    expires      DateTime
-    user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+  id           String   @id @default(auto()) @map("_id") @db.ObjectId
+  sessionToken String   @unique
+  userId       String   @db.ObjectId
+  expires      DateTime
+  user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
 }
 
 model User {
-    id            String    @id @default(auto()) @map("_id") @db.ObjectId
-    name          String?
-    email         String?   @unique
-    emailVerified DateTime?
-    image         String?
-
-    // @deprecated Pending for removal when we expand to multiple universities
-    admin         Boolean   @default(false)
-    role          Role      @default(USER)
-
-    Account       Account[]
-    Session       Session[]
-    SeniorIDs     String[]  @db.ObjectId
-    Seniors       Senior[]  @relation(fields: [SeniorIDs], references: [id])
-    approved      Approval  @default(PENDING)
+  id            String    @id @default(auto()) @map("_id") @db.ObjectId
+  name          String?
+  firstName     String?
```

Comment:
I would opt-in for [default values](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model#defining-a-default-value) instead of optional. It makes the consumer code a bit cleaner.